install/0000_30_machine-api-operator_08_rbac.yaml

```diff
@@ -1,13 +1,78 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: cluster-api-manager-role
```

Comment:
This is a nit, but I don't think these names should end in `-role` etc. You already know that by the type, and it looks weird in the console. I would just call this `machine-api` or something.